st/CMSIS_STM32WBAxx/Device/ST/STM32WBAxx/.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+---

```

Comment:
The files in the .github directory should not be checked in

Reply:
For sure!